ruby/app/models/Verification.rb

```diff
@@ -0,0 +1,10 @@
+class Verification
+  def self.get(public_token)
+    access_token = Citadel.getAccessToken(public_token)
+    if ENV.fetch("API_PRODUCT_TYPE") == "employment"
```

Comment:
Same. `ENV.fetch` shouldn't be used outside initializers

Reply:
corrected as per suggestion